src/searchModal/advancedSearch.js

```diff
@@ -131,30 +134,45 @@ export default function advancedSearchFactory(config) {
      * Inits select2 on criteria select and its UX logic
      */
     function initAddCriteriaSelector() {
-        $criteriaSelect.select2({
-            containerCssClass: 'criteria-select2',
-            dropdownCssClass: 'criteria-dropdown-select2',
-            sortResults: results => _.sortBy(results, ['text'])
-        });
-
-        // open dropdown when user clicks on add criteria input
-        $addCriteriaInput.on('click', () => {
-            $criteriaSelect.select2('open');
-            // if dropdown is opened above addCriteria input, top property is slightly decreased to avoid overlapping with addCriteria icon
-            if ($('.criteria-dropdown-select2').hasClass('select2-drop-above')) {
-                $('.criteria-dropdown-select2').css(
-                    'top',
-                    $('.criteria-dropdown-select2').css('top').split('px')[0] - 10 + 'px'
-                );
+        const route = urlUtil.route('status', 'advanced_search', 'tao');
```

Comment:
Route will be `tao/AdvancedSearch/status` 😄

https://github.com/oat-sa/tao-core/pull/2719/files#diff-7ef472c51d586c321f4bfb5ee093c8ad8b70f31bebf72bd2042b4a5021de0c48R27

Reply:
Done, thanks